src/fidesops/service/connectors/post_processor_strategy/post_processor_strategy_unwrap.py

```diff
@@ -0,0 +1,69 @@
+import logging
+from typing import Any, Optional, Dict
+
+from fidesops.schemas.saas.strategy_configuration import (
+    UnwrapPostProcessorConfiguration,
+    StrategyConfiguration,
+)
+from fidesops.service.connectors.post_processor_strategy.post_processor_strategy import (
+    PostProcessorStrategy,
+)
+
+
+STRATEGY_NAME = "unwrap"
+
+logger = logging.getLogger(__name__)
+
+
+class UnwrapPostProcessorStrategy(PostProcessorStrategy):
+    """
+    Given a path to a dict, returns the dict/list
+    E.g.
+    data = {
+        exact_matches: {
+            members: [
+                howdy: 123,
+                meow: 841
+            ]
+        }
+    }
+    data_path = exact_matches.members
```

Comment:
The more I think about this `data_path` param from the saas config, it feels like we're using two fields instead of one to target the desired information.  Like you need the `data_path` to get part of the way there, and then the `field_name` to get the rest of it.   In the mongo_yaml for example, we just have more complex references in one place: 

```yaml
...
fields:
              - name: internal_id
                fidesops_meta:
                  data_type: string
                  references:
                    - dataset: mongo_test
                      field: customer_feedback.customer_information.internal_customer_id
                      direction: from
```

Reply:
wait I'm not following. The `data_path` param should include all the data needed. For `exact_matches.members`, that gives us all the info we need to access the `members` obj